runtime/jvmti/jvmtiHeap.c

```diff
@@ -1250,6 +1250,7 @@ mapEventType(J9JVMTIHeapData * data, IDATA type, jint index, j9object_t referrer
 		case J9GC_ROOT_TYPE_CLASSLOADER:
 		case J9GC_ROOT_TYPE_JVMTI_TAG_REF:
 		case J9GC_REFERENCE_TYPE_UNKNOWN:
+		case J9GC_REFERENCE_TYPE_FCC:
```

Comment:
@dmitripivkine just to confirm, we want it to fall through and hit the `J9JVMTI_HEAP_EVENT_NONE_NOFOLLOW;` right?

Reply:
Yes, I think we can do this